app/assets/javascripts/discourse/app/components/composer-editor.js

```diff
@@ -203,18 +204,19 @@ export default class ComposerEditor extends Component {
 
     const input = this.element.querySelector(".d-editor-input");
 
-    input?.addEventListener(
-      "scroll",
-      this._throttledSyncEditorAndPreviewScroll
-    );
+    input.addEventListener("scroll", this._throttledSyncEditorAndPreviewScroll);
```

Comment:
Unrelated change, but if we get to this point without a defined `input`, we should fix it